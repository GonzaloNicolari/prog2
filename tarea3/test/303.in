# prueba de izquierdo, derecho, altura_binario, imprimir_binario, buscar_subarbol
altura_binario
imprimir_binario
buscar_subarbol a
insertar_en_binario (8,r)
altura_binario
imprimir_binario
izquierdo
derecho
buscar_subarbol a
buscar_subarbol r
insertar_en_binario (6,y)
altura_binario
imprimir_binario
izquierdo
derecho
buscar_subarbol a
buscar_subarbol r
buscar_subarbol y
insertar_en_binario (6,t)
altura_binario
insertar_en_binario (4,m)
altura_binario
imprimir_binario
izquierdo
derecho
insertar_en_binario (4,g)
insertar_en_binario (4,d)
insertar_en_binario (4,h)
insertar_en_binario (4,i)
insertar_en_binario (4,j)
altura_binario
imprimir_binario
izquierdo
derecho
buscar_subarbol t
buscar_subarbol f
buscar_subarbol j
buscar_subarbol d
buscar_subarbol y
Fin



