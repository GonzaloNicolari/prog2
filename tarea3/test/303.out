1>#  prueba de izquierdo, derecho, altura_binario, imprimir_binario, buscar_subarbol.
2>0.
3>
4>sub es vacío.
5>Insertado.
6>1.
7>
(8,r)
8>
9>
10>sub es vacío.
11>(8,r)
12>Insertado.
13>2.
14>
-(6,y)
(8,r)
15>
16>
(6,y)
17>sub es vacío.
18>(8,r)
19>(6,y)
20>Insertado.
21>3.
22>Insertado.
23>3.
24>
-(6,y)
--(6,t)
(8,r)
-(4,m)
25>
(4,m)
26>
(6,y)
-(6,t)
27>Insertado.
28>Insertado.
29>Insertado.
30>Insertado.
31>Insertado.
32>6.
33>
-(6,y)
--(6,t)
(8,r)
-(4,m)
-----(4,j)
----(4,i)
---(4,h)
--(4,g)
---(4,d)
34>
(4,m)
----(4,j)
---(4,i)
--(4,h)
-(4,g)
--(4,d)
35>
(6,y)
-(6,t)
36>(6,t)
37>sub es vacío.
38>(4,j)
39>(4,d)
40>(6,y)
41>Fin.
